libnavigation-core/src/test/java/com/mapbox/navigation/core/reroute/MapboxRerouteControllerTest.kt

```diff
@@ -446,6 +447,32 @@ class MapboxRerouteControllerTest {
         return rerouteController.registerRerouteStateObserver(rerouteStateObserver)
     }
 
+    @Test
+    fun uses_route_options_delegate() {
+        mockRouteOptionsResult(successFromResult)
+        val mockNewRouteOptions = mockk<RouteOptions>()
+        val mockRerouteOptionsDelegateManger = mockk<RerouteOptionsAdapter> {
```

Comment:
```suggestion
        val mockRerouteOptionsDelegateManager = mockk<RerouteOptionsAdapter> {
```
nit

Reply:
I would not change that to avoid long waiting for CI checks